build_library/grub.cfg

```diff
@@ -57,23 +72,25 @@ if [ "$net_default_server" != "" ]; then
 fi
 
 # Search for the OEM partition, load additional configuration if found.
-if [ "$secure_boot" = "0" ]; then
+if [ "$secure_boot" = "0" -a -z "$isoboot" ]; then
```

Comment:
Does this form of and actually work in bash? I know in the past we've switched to a nested if because and didn't work but that was probably using a different syntax.